va/va_dec_hevc.h

```diff
@@ -194,6 +194,154 @@ typedef struct  _VAPictureParameterBufferHEVC
+            uint32_t        high_precision_offsets_enabled_flag         : 1;
+            uint32_t        persistent_rice_adaptation_enabled_flag     : 1;
+            uint32_t        cabac_bypass_alignment_enabled_flag         : 1;
+            uint32_t        cross_component_prediction_enabled_flag     : 1;
+            uint32_t        chroma_qp_offset_list_enabled_flag          : 1;
+
+            uint32_t        reserved_bits                               : 21;
+        } bits;
+        uint32_t            value;
+    } range_extension_pic_fields;
+
+    /** \brief HEVC range extension flags
+     *  The following flags bears the same syntax and semantics as
+     *  those defined in HEVC bit stream spec.
+     */
+    uint8_t                 diff_cu_chroma_qp_offset_depth;
+    uint8_t                 chroma_qp_offset_list_len_minus1;
+    uint8_t                 log2_sao_offset_scale_luma;
+    uint8_t                 log2_sao_offset_scale_chroma;
+    uint8_t                 log2_max_transform_skip_block_size_minus2;
+    int8_t                  cb_qp_offset_list[6];
+    int8_t                  cr_qp_offset_list[6];
+} VAPictureParameterBufferHEVCRext;
+
+/**
+ *\brief HEVC Decoding Picture Parameter Buffer Structure for
+ *Screen Content extension
+ *
+ *This structure conveys picture level HEVC Scc parameters
+ *and should be sent once per frame. This data structure should be sent
+ *together with VAPictureParameterBufferHEVC.
+ *
+ */
+typedef struct  _VAPictureParameterBufferHEVCScc
+{
+    union
+    {
+        struct
+        {
+            /** \brief HEVC Scc extension flags
+             *  The following flags bears the same syntax and semantics as
+             *  those defined in HEVC bit stream spec.
+             */
+            /*  indicates if intra block copy (IBC) is enabled or not. */
+            uint32_t        pps_curr_pic_ref_enabled_flag                   : 1;
+            /*  indicates if Palette Mode is enabled or not. */
+            uint32_t        palette_mode_enabled_flag                       : 1;
+            /*  controls the presence and inference of the use_integer_mv_flag syntax
+             *  in slice segment header that specifies the resolution of motion
+             *  vectors for inter prediction.
+             */
+            uint32_t        motion_vector_resolution_control_idc            : 2;
+            /*  specifies that the intra boundary filtering process is
+             *  disabled or not for intra prediction.
+             */
+            uint32_t        intra_boundary_filtering_disabled_flag          : 1;
+            /*  specifies that an adaptive colour transform may be applied
+             *  to the residual in the decoding process.
+             */
+            uint32_t        residual_adaptive_colour_transform_enabled_flag : 1;
+
+            uint32_t        pps_slice_act_qp_offsets_present_flag           : 1;
```

Comment:
You added comments for some of these flags, could you add comments for other flags?